### DISCUSSION TITLE
If adjust the parameters and add Lora model can solve the resolution？

### DISCUSSION THREAD

**Discussion by xtiondiv123** [lllyasviel/Fooocus#2869] [Q&A] [answered]
Hi, all. This is the question from high-resolution from the part refiner.
1 this picture is from a floor photo which is a real photo by camera.  Size is more than 10M
![1714856231871](https://github.com/lllyasviel/Fooocus/assets/152345887/62df074e-2c6f-4bce-adc8-a4e336d8c23b)
2 refiner part is low resolution ,and  i am not sure if  I miss my lora model . 
![1714856346694](https://github.com/lllyasviel/Fooocus/assets/152345887/a98021bf-2b4a-40e9-921f-96d5265d750a)
this final picture which only used on web , can not print. i  want to make resolution of refiner part be same as the floor part.
this is my log here :
![1714856844506](https://github.com/lllyasviel/Fooocus/assets/152345887/b6bd6461-9b52-4ef5-8753-4e39321b54fb)
Do I need to adjust the parameters like Guidancescale 、Sharpenss、 Adm guidance 、sampler scheduler、 and add the Lora model to improve the resolution of the image?
Thank you all， wait online .

**Comment by mashb1t** [comment, marked as answer]
If you're happy with the composition, please check out the Fooocus upscaling methods, see https://github.com/lllyasviel/Fooocus/discussions/390

Your resolution of 960x1024 seems to be odd though and not a standard resolution of SDXL. Find more in https://github.com/lllyasviel/Fooocus/pull/1617

**Reply by xtiondiv123** [reply]
I got it，i will change the size thank you very much
